### PR TITLE
Awesome Book Pair programming Project

### PR DESCRIPTION
**Pull request by Nicolaswg**
> In this project we are working with JS DOM manipulation and Browser localStorage.

### Main Point

- Create dynamically element inside the Dom.
- Store and use the date from the browser localStorage.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      const buttons = document.querySelectorAll('button.remove');
+      buttons.forEach((e, i) => {
+        buttons[i].addEventListener(
+          'click',
+          (event) => {
+            removeBook(arrBooks.indexOf(arrBooks[i]));
+            event.preventDefault();
+          },
+        );
+      });
```
Comment (MrBrN197) [inline main.js]: - [x] Hi, Nice job with your code for adding new books. Please refactor this functionality into a separate function named something like `addBooks`.

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @danielufeli & @Nicolaswg 👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!
## To highlight 👏👏
- ✔Code functionality is working correctly
- ✔ Professional README

## Required Changes ♻️
- [x] Please add a package.json file with the linter dependencies that are required by your project.
- [x] Please add a .gitignore file and add the /node_modules folder to it.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
+/* eslint-disable no-use-before-define */
```
Comment (MrBrN197) [inline main.js]: - [ ] Please avoid disabling ESlint to get around linter warnings problems. These ESlint warnings are there to ensure you adhere to correct coding standards and when you have warnings they usually signal an underlying issue with how you've written your code.

File: main.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    bookContent += `<p class="title">${arrBooks[i].title}</p>
+  <p class="author">${arrBooks[i].author}</p>
+  <button class="remove">Remove</button>
+  <div class="line"></div>`;
+    document.querySelector('.books').innerHTML = bookContent;
+    const buttons = document.querySelectorAll('button.remove');
+    const removeBook = (index) => {
+      arrBooks = arrBooks.filter((e, i) => i !== index);
+      storeBooks(JSON.stringify(arrBooks));
+      // eslint-disable-next-line no-use-before-define
```
Comment (MrBrN197) [inline main.js]: - [ ] Please avoid disabling ESlint to get around linter warnings problems. These ESlint warnings are there to ensure you adhere to correct coding standards and when you have warnings they usually signal an underlying issue with how you've written your code.

File: main.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+      return displayBooks();
+    };
+    buttons.forEach((e, i) => {
+      buttons[i].addEventListener('click', (event) => {
+        // eslint-disable-next-line no-undef
```
Comment (MrBrN197) [inline main.js]: - [ ] Please avoid disabling ESlint to get around linter warnings problems. These ESlint warnings are there to ensure you adhere to correct coding standards and when you have warnings they usually signal an underlying issue with how you've written your code.

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @Nicolaswg  & @danielufeli 👋 

Thank you for making my requested changes. I've noticed you've introduced some eslint-disables please avoid doing this.
You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 2 above)
Comment (Nicolaswg) [reply main.js]: OK, thanks I'm just wondering about calling back a function before it is created so if I do that eslint make an error and that that's the function that you said to change ...

Comment (MrBrN197) [reply main.js]: @Nicolaswg  make sure to declare the function before you use it. that's what the ESlint error is referring to.

**Review by descholar-ceo — APPROVED** [review]
Hi @Nicolaswg and @danielufeli , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**